chess-AI/train.py

```diff
@@ -214,8 +215,9 @@ def train_on_dataset(dataset, nnet, options, show_dash=False):
         Dashboard.visualize_losses(average_pol_loss, average_val_loss)
 
     # Saves model to specified file, or a new file if not specified.
-    # TODO: Figure frequency of model saving, right now it is after a defined number of epochs.
-    save_model(nnet, options.save_path, options.num_saved_models, options.overwrite)
+    if save:
+        save_model(nnet, options.save_path + "/model_{}".format(iteration), options.num_saved_models,
```

Comment:
This might give problems on windows if we use unix style paths. Maybe it would be best to use [pathlib](https://docs.python.org/3/library/pathlib.html) here

Reply:
hasn't been giving me any problems, torch.save probably uses pathlib internally on string input